src/main/java/seedu/fridgefriend/utilities/Parser.java

```diff
@@ -33,12 +33,12 @@
      * @param input user's raw input
      * @return Command object
      * @throws EmptyDescriptionException if the required description field is empty
-     * @throws InvalidInputException if the command is not recognised
-     * @throws InvalidIndexException if the index given in description is out of bounds
-     * @throws InvalidDateException if the date input cannot be parsed
+     * @throws InvalidInputException     if the command is not recognised
```

Comment:
Is extra space allow here?

Reply:
It looks nice on my compiler, can double check?

![image](https://user-images.githubusercontent.com/2010527/111948348-f96c4e00-8b19-11eb-9cca-8e71a06b694e.png)

Reply:
i think they trim by default, best to just remove the extra spaces